### ISSUE TITLE
LSEPLoss  didn't decrease 

### ISSUE THREAD

**Issue by YangQun1** [Mipanox/Bird_cocktail#1]
I use the LSEPLoss implemented in this repo to train a multilabel image classification CNN, but I found that the loss didn't decrease. And, when I replace the LSEPLoss to bce loss, the training process goes well. 
I tune the learning rate to 1e-3, 1e-4 and 1e-5, but none of them worked.
So, I want to ask, is there something needed to be pay attention when using le lesp loss?

My pytorch version is 0.4.1, label number is about 1000, and the backbone is resnet50.
